tests/icons.test.js

```diff
@@ -25,8 +30,14 @@ icons.forEach((icon) => {
     expect(subject.source).toEqual(icon.source);
   });
 
-  test(`${icon.title} has an "svg" value`, () => {
+  test(`${icon.title} has a valid "svg" value`, () => {
     expect(typeof subject.svg).toBe('string');
+    const svgFileContents = fs
+      .readFileSync(svgPath, 'utf8')
+      .replace(/\r?\n/, '');
+    expect(subject.svg.substring(subject.svg.indexOf('<title>'))).toEqual(
+      svgFileContents.substring(svgFileContents.indexOf('<title>')),
+    );
```

Comment:
Shouldn't the `.svg` content and the `subject.svg` file not just equal eachother?

Reply:
The attributes `role`, `viewBox` and `xmlns` are not ordered in all SVGs, but in the template now there are.

Reply:
:unamused:

Wonder if that makes it worth it for us to enforce the ordering through SVGLint, so that we can update all the SVGs and then these tests :thinking: Definitely outside the scope of this Pull Request, but it could be worth opening a ticket if you agree.

Reply:
Opened #6826 to track it 👍🏼